README.md

```
@@ -0,0 +1,143 @@
+    - requires jmp-dto
+    - export Service interface
+
+##### 11. Implement Service into jmp-cloud-service-impl. Use Stream API, You can use Memory/DB for data storing :
+
+##### 12. Add module-info.java which :
+
+    - requires transitive module with Service interface
+    - requires jmp-dto
+    - exports implementation of Service interface
+
+##### 13. Use var for definition of local variables wherever it’s applicable.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)